src/ApiInterface/include/ApiInterface/Orbit.h

```diff
@@ -350,7 +350,13 @@ extern ORBIT_EXPORT void* orbit_api_get_function_table_address_v1();
   struct orbit_api_v1 g_orbit_api_v1; \
   void* orbit_api_get_function_table_address_v1() { return &g_orbit_api_v1; }
 
-inline bool orbit_api_active() {
+#ifndef __cplusplus
+// In C, `inline` alone doesn't generate an out-of-line definition, causing a linker error if the
+// function is not inlined. We need to make the function `static`.
+static
+#endif
+    inline bool
```

Comment:
If I'm not mistaken `extern inline` should work in both C++ and C the same way and should achieve
what you want without having an `ifdef`. But I could be wrong.

Reply:
Hehe that is the reason why I added two files instead of one. The problem is that in C `inline` doesn't mean "multiple definitions allowed". In other words `inline` is kind of useless. So `extern inline` generates an externally visible function just like omitting both, which results in multiple definitions and hence a linking error.

Reply:
Ah I see. Thanks for the explanation.